tools/mmp/driver.cs

```diff
@@ -1374,14 +1387,22 @@ static void LipoLibrary (string name, string dest)
 			if (existingArchs.Count () < 2)
 				return;
 
-			var arch = App.Abi.AsString ();
 			// macOS frameworks often uses symlinks and we do not
 			// want to replace the symlink with the thin binary
 			// while leaving the fat binary inside the framework
 			dest = GetRealPath (dest);
-			RunLipo (App, new [] { dest, "-thin", arch, "-output", dest });
-			if (name != "MonoPosixHelper" && name != "libmono-native-unified" && name != "libmono-native-compat")
-				ErrorHelper.Warning (2108, Errors.MM2108, name, arch);
+
+			var prefix = new [] { dest };
+			var suffix = new [] { "-output", dest };
+			List <string> archArgs = new List <string> ();
+			foreach (var abi in App.Abis)
+			{
```

Comment:
minor: `{` should be on the same line as `foreach`

Reply:
Fixed at bc0558c1092c56fa09317a4fa7ee509ff1bf28ba